pulsar-client-admin/src/main/resources/mltool/ManagedLedgerAdminTool.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python
```

Comment:
Should we just place this scripts under `bin/python/...` ?
And include this in the src/bin distribution tgz.

At the same time, let's just use a similar naming as the other CLI tools. Eg: 

`bin/pulsar-managed-ledger-admin`